tests/instruments/test_common_base.py

```diff
@@ -711,3 +719,18 @@ def test_dynamic_property_values_update_in_one_instance_leaves_other_unchanged()
 def test_dynamic_property_reading_special_attributes_forbidden(fake):
     with pytest.raises(AttributeError):
         fake.fake_ctrl_validator
+
+
+def test_dynamic_property_with_inheritance():
+    inst = ExtendedBase()
+    # Test for inherited attribute
+    with pytest.raises(AttributeError):
+        inst.fake_ctrl_validator
+    # Test for new attribute
+    with pytest.raises(AttributeError):
+        inst.fake_ctrl2_validator
+
+
+def test_dynamic_property_values_defined_at_superclass_level():
+    inst = StrictExtendedBase()
+    inst.fake_ctrl2 = 17  # outside original values (1, 10)
```

Comment:
This test doesn't assert anything?

Reply:
The assignment would fail, if the values parameter would not have been changed from the original values (1, 10) to (2, 20).
Not failing is passing.
And I failed in making the comment understandable...